cmd/kaspawallet/transactions_hex_encoding.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"encoding/hex"
+	"strings"
+)
+
+// hexTransactionsSeparator is used to mark the end of one transaction and the beggining of the next one.
+// We use a separator that is not in the hex alphabet, but which will not split selection with a double click
+const hexTransactionsSeparator = "_"
+
+func encodeTransactionsToHex(transactions [][]byte) string {
+	transactionsInHex := make([]string, len(transactions))
+	for i, transaction := range transactions {
+		transactionsInHex[i] = hex.EncodeToString(transaction)
+	}
+	return strings.Join(transactionsInHex, hexTransactionsSeparator)
+}
+
+func decodeTransactionsFromHex(transactionsHex string) ([][]byte, error) {
```

Comment:
nit: I think it's best to use a protobuf type that will hold an array of transactions, instead of parsing with separator manually